OpenRA.Mods.Common/Widgets/WorldInteractionControllerWidget.cs

```diff
@@ -241,7 +241,8 @@ public override bool HandleKeyPress(KeyInput e)
 			{
 				var key = Hotkey.FromKeyInput(e);
 
-				if (key == Game.Settings.Keys.PauseKey && World.LocalPlayer != null) // Disable pausing for spectators
+				if (key == Game.Settings.Keys.PauseKey && World.LocalPlayer != null &&
+					(Game.IsHost || player.WinState != WinState.Lost)) // Disable pausing for spectators and defeated players
```

Comment:
Changing this to `key == Game.Settings.Keys.PauseKey && (Game.IsHost || (World.LocalPlayer != null && World.LocalPlayer.WinState != WinState.Lost))` should in principle address both points (but do test this, because it may uncover other issues).